content/docs/typechecking-with-proptypes.md

```diff
@@ -30,11 +30,11 @@ Greeting.propTypes = {
 };
 ```
 
-`PropTypes` exports a range of validators that can be used to make sure the data you receive is valid. In this example, we're using `PropTypes.string`. When an invalid value is provided for a prop, a warning will be shown in the JavaScript console. For performance reasons, `propTypes` is only checked in development mode.
+`PropTypes` mengirimkan berbagai jenis validator yang dapat digunakan untuk memastikan bahwa data yang diterima valid. Contoh diatas, kita menggunakan `PropTypes.string`. Ketika nilai yang dikirimkan untuk sebuah *prop* keliru, sebuah peringatan akan muncul di konsol JavaScript. Untuk alasan performa, `propTypes` hanya melakukan pengecekan di mode pengembangan atau *development*.
 
 ### PropTypes {#proptypes}
 
-Here is an example documenting the different validators provided:
+Berikut contoh mendokumentasikan berbagai validator yang berbeda yang sudah disediakan:
```

Comment:
validator -> *validator*

Reply:
diatas -> di atas

I think
"Untuk alasan performa, `propTypes` hanya melakukan pengecekan di mode pengembangan."
is enough

What do you think? 😄